### PR TITLE
Allow for a "base" use of box without needing to change the defaults of vagrant with virtualbox

### LINKED ISSUE #84: Vagant box init without varantfile should work
`vagrant init fredrikhgrelland/hashistack`
`vagrant up`
 should be meaningful

### LINKED ISSUE #88: Remove reliance on additional network.
Rethinking if we really need eth1. If not THIS is the time to change it.

_Originally posted by @fredrikhgrelland in https://github.com/fredrikhgrelland/vagrant-hashistack/issues/84#issuecomment-648736966_

The reasons for doing this was to have a stable ip address to reference, but it is a pain you need to change networking to get basic services up and running. Removing eth1 outright, and we may use static ip in vagrant file if needed.

### PR DESCRIPTION
**Pull request by fredrikhgrelland**
close #88 
close #84 

Introduces a breaking change to fredrikhgrelland/data-mesh etc. now or never.

### CODE REVIEW

**Comment by fredrikhgrelland** [comment]
Hmm. It works, but I am currently not able to reach ports. Working on it, but if I am not able to fix it I'll have to regress.

**Comment by fredrikhgrelland** [comment]
> Hmm. It works, but I am currently not able to reach ports. Working on it, but if I am not able to fix it I'll have to regress.

Not worth it. Will require more hacks.

File: test/Vagrantfile [hunk 1]
```diff
@@ -3,11 +3,9 @@ Vagrant.configure("2") do |config|
     config.vm.network "private_network", ip: "10.0.3.10"
     config.vm.box_url = "file://../packer/output-hashistack/package.box"
 
-    # Hashicorp consul ui
+    # Hashistack
```
Comment (zhenik) [inline test/Vagrantfile:6]: I think previous comment `# Hashicorp consul ui` had a better description

**Comment by fredrikhgrelland** [comment]
> What about a section to README on how to get a box up and running without a vagrantfile existing?

The Usage section should be sufficient, but with templates we can rewrite this.

File: test/Vagrantfile (hunk 1 above)
Comment (fredrikhgrelland) [reply test/Vagrantfile]: This is now a part of default box

**Comment by fredrikhgrelland** [comment]
@zhenik If you think this is ready, I'll push a changelog on this and make a patch release out of it so we can continue on #93

**Pull request merged**